modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -149,7 +151,14 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
             'Links',
             'sessionID'
         );
-       
+
+        // Insert project column after DCCID, if useProject config is enabled
+        if ($config->getSetting('useProjects') === "true") {
```

Comment:
Nevermind, I didn't realize that useProjects is from the database, not the config.xml when I wrote my suggestion. It should be consistent with just a "true" check.